Fix rocsp unit test flakes

```
D161252 rocsp-tool.test 88b-5wc SQL:  insert into `certificateStatus` (`id`,`serial`,`status`,`ocspLastUpdated`,`revokedDate`,`revokedReason`,`lastExpirationNagSent`,`ocspResponse`,`notAfter`,`isExpired`,`IssuerID`) values (null,?,?,?,?,?,?,?,?,?,?); [1:"1337" 2:"" 3:0001-01-01 00:00:00 +0000 UTC 4:0001-01-01 00:00:00 +0000 UTC 5:0 6:0001-01-01 00:00:00 +0000 UTC 7:[] 8:1970-01-01 12:00:00 +0000 UTC 9:false 10:0] (1.433177ms)
D161252 rocsp-tool.test qYyKowg SQL:  insert into `certificateStatus` (`id`,`serial`,`status`,`ocspLastUpdated`,`revokedDate`,`revokedReason`,`lastExpirationNagSent`,`ocspResponse`,`notAfter`,`isExpired`,`IssuerID`) values (null,?,?,?,?,?,?,?,?,?,?); [1:"1338" 2:"" 3:0001-01-01 00:00:00 +0000 UTC 4:0001-01-01 00:00:00 +0000 UTC 5:0 6:0001-01-01 00:00:00 +0000 UTC 7:[] 8:1970-01-02 12:00:00 +0000 UTC 9:false 10:0] (988.953µs)
--- FAIL: TestStoreResponse (0.12s)
    client_test.go:115: storing response: storing response: transaction failed: got 4 elements in cluster info address, expected 2 or 3
FAIL
FAIL	github.com/letsencrypt/boulder/cmd/rocsp-tool	0.543s
```

and

```
--- FAIL: TestSetAndGet (0.01s)
    rocsp_test.go:50: storing response: transaction failed: got 4 elements in cluster info address, expected 2 or 3
FAIL
FAIL	github.com/letsencrypt/boulder/rocsp	0.042s
```

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -66,7 +66,7 @@ services:
         driver: none
 
   bredis_1:
-    image: redis:latest
+    image: redis:6.2.7
     volumes:
       - ./test/:/test/:cached
     command: redis-server /test/redis.config
@@ -75,7 +75,7 @@ services:
           ipv4_address: 10.33.33.2
 
   bredis_2:
-    image: redis:latest
+    image: redis:6.2.7
     volumes:
       - ./test/:/test/:cached
     command: redis-server /test/redis.config
@@ -84,7 +84,7 @@ services:
           ipv4_address: 10.33.33.3
 
   bredis_3:
-    image: redis:latest
+    image: redis:6.2.7
     volumes:
       - ./test/:/test/:cached
     command: redis-server /test/redis.config
@@ -93,7 +93,7 @@ services:
           ipv4_address: 10.33.33.4
 
   bredis_4:
-    image: redis:latest
+    image: redis:6.2.7
     volumes:
       - ./test/:/test/:cached
     command: redis-server /test/redis.config
@@ -102,7 +102,7 @@ services:
           ipv4_address: 10.33.33.5
 
   bredis_5:
-    image: redis:latest
+    image: redis:6.2.7
     volumes:
       - ./test/:/test/:cached
     command: redis-server /test/redis.config
@@ -111,7 +111,7 @@ services:
           ipv4_address: 10.33.33.6
 
   bredis_6:
-    image: redis:latest
+    image: redis:6.2.7
     volumes:
       - ./test/:/test/:cached
     command: redis-server /test/redis.config
@@ -120,7 +120,7 @@ services:
           ipv4_address: 10.33.33.7
 
   bredis_clusterer:
-    image: redis:latest
+    image: redis:6.2.7
     volumes:
       - ./test/:/test/:cached
       - ./cluster/:/cluster/:cached

